types/ts3.9.4/index.d.ts

```diff
@@ -91,22 +93,18 @@ export const Switch: FunctionComponent<SwitchProps>;
  * Components: <Router />
  */
 
-export interface RouterProps {
-  hook: BaseLocationHook;
-  base: Path;
-  matcher: MatcherFn;
-}
-export const Router: FunctionComponent<
-  Partial<RouterProps> & {
-    children: ReactNode;
-  }
->;
+export type RouterProps = Partial<RouterOptions> & {
+  parent?: RouterObject;
```

Comment:
Shouldn't we move `parent` into `RouterOptions`? This would allow us to do `useRouter().parent`.

Reply:
👍 Fixed in [1843f4c](https://github.com/molefrog/wouter/pull/274/commits/1843f4cd33f46c65fc9adfe4de96d626532e77e1)